spec/system/task_spec.rb

```diff
@@ -46,69 +46,61 @@
     context '正常系' do
       it 'Taskが表示されること' do
         # TODO: ローカル変数ではなく let を使用してください
-        project = FactoryBot.create(:project)
-        task = FactoryBot.create(:task, project_id: project.id)
-        visit project_task_path(project, task)
+        visit project_task_path(task.project, task)
         expect(page).to have_content(task.title)
         expect(page).to have_content(task.status)
         expect(page).to have_content(task.deadline.strftime('%Y-%m-%d %H:%M'))
-        expect(current_path).to eq project_task_path(project, task)
+        expect(current_path).to eq project_task_path(task.project, task)
       end
     end
   end
 
   describe 'Task編集' do
     context '正常系' do
-      xit 'Taskを編集した場合、一覧画面で編集後の内容が表示されること' do
+      it 'Taskを編集した場合、一覧画面で編集後の内容が表示されること' do
         # FIXME: テストが失敗するので修正してください
-        project = FactoryBot.create(:project)
-        task = FactoryBot.create(:task, project_id: project.id)
-        visit edit_project_task_path(project, task)
+        visit edit_project_task_path(task.project, task)
         fill_in 'Deadline', with: Time.current
         click_button 'Update Task'
         click_link 'Back'
-        expect(find('.task_list')).to have_content(Time.current.strftime('%Y-%m-%d'))
-        expect(current_path).to eq project_tasks_path(project)
+        expect(find('.task_list')).to have_content short_time(Task.first.deadline)
```

Comment:
`Task.first`はテストケース実施に当たってテストデータが余計に作成されると破綻するので、なるべく避けた方が良いです！
今回は`task`に`reload`を使うと更新後の状態で内容が取得できるので利用できるかと思います。
```suggestion
        expect(find('.task_list')).to have_content short_time(task.reload.deadline)
```

Reply:
そんな機能あるんですね！！知りませんでした！！
a71020a で修正しました。